### PR TITLE
OS-888 fix: Import Repo address from plugin-info

### PR DESCRIPTION
**Pull request by Rekard0**

### CODE REVIEW

File: packages/subgraph/package.json [hunk 1]
```diff
@@ -20,14 +20,15 @@
     "clean": "rimraf deploy-output.txt subgraph.yaml ./build ./imported ./generated ./tests/.bin tests/.latest.json && yarn postinstall"
   },
   "devDependencies": {
-    "osx-ethersV120": "npm:@aragon/osx-ethers@1.2.1",
     "@graphprotocol/graph-cli": "^0.51.0",
     "@graphprotocol/graph-ts": "^0.31.0",
     "cross-env": "^7.0.3",
     "matchstick-as": "^0.5.2",
     "mustache": "^4.2.0",
+    "osx-ethersV120": "npm:@aragon/osx-ethers@1.2.1",
```
Comment (heueristik) [inline packages/subgraph/package.json]:
```suggestion
```
I think you included this in the beginning and then realized that you find the repo in the `plugin-info.json`. At least I don't see it used anywhere.

File: packages/subgraph/scripts/import-plugin-repo.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import dotenv from 'dotenv';
+import fs from 'fs';
+import path from 'path';
+
+// Specify the path to the .env file at the root
+const rootDir = path.join(__dirname, '../../../'); // Adjust the path as necessary
+dotenv.config({path: path.join(rootDir, '.env')});
+
+// Extract Repo address from the contracts/plugin-info.json
```
Comment (heueristik) [inline packages/subgraph/scripts/import-plugin-repo.ts]: Have you seen [`packages/contracts/utils/helpers.ts`](https://github.com/aragon/osx-plugin-template-hardhat/blob/d3af915db62f9681d79d83a018c0dfb601652634/packages/contracts/utils/helpers.ts#L40-L48)?
Is this an assembly script limitation?
If possible, let's not duplicate more things 🙏

File: packages/subgraph/package.json (hunk 1 above)
Comment (josemarinas) [reply packages/subgraph/package.json]: It is used here `packages/subgraph/scripts/postInstall.ts`

File: packages/subgraph/scripts/import-plugin-repo.ts (hunk 2 above)
Comment (josemarinas) [reply packages/subgraph/scripts/import-plugin-repo.ts]: yes, this at some point should be encapsulñated in the sdk-commons

File: packages/subgraph/package.json (hunk 1 above)
Comment (Rekard0) [reply packages/subgraph/package.json]: then we need to update to 1.3.0

**Review by mathewmeconry — APPROVED** [review]
beside the comment about updating osx-ethers to v1.3.0 it looks good to me

File: .gitignore [hunk 3]
```diff
@@ -21,7 +21,7 @@ packages/subgraph/tests/.bin
 .pnp.*
 coverage.json
 
-packages/contracts/plugin-info.json
+plugin-info.json
```
Comment (Rekard0) [inline .gitignore]: r we sure we want to gitignore this file ? @Michael-A-Heuer

Comment (heueristik) [inline .gitignore]:
```suggestion
plugin-info-testing.json
```
This one should be ignored.

File: packages/subgraph/scripts/postInstall.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
-import fs from 'fs';
 // Import the osx-ethers module to access OSX contracts and generate their ABIs for any needed version.
-import * as osx120 from 'osx-ethersV120';
+import * as osx120 from '@aragon/osx-ethers';
```
Comment (heueristik) [inline packages/subgraph/scripts/postInstall.ts]: please change as well

**Review by heueristik — APPROVED** [review]

**Review by heueristik — APPROVED** [review]

**Pull request merged**